integrations/extensions/starter-kits/language-model-summarization/README.md

```diff
@@ -0,0 +1,75 @@
+
+1. Enter something unrelated, such as `I want to close my bank account`.
+
+1. Enter something funny, such as `It was the path to freedom`.
+
+1. To complete the conversation, enter something like `Tell me why`. The assistant says `To get to the other side!` and the action ends.
+
+1. To see the summary, enter `human agent`. This causes Watson Assistant to invoke the `Fallback` action, which is configured to call the `Escalate to Agent` action.  This then triggers the `Invoke GPT Chat Completion API` action and shows the response that is returned.
+
+An example response might look something like this: `The user asked the chatbot to tell a joke. The chatbot asked "Why did the chicken cross the road?" but the user talked about liking chicken salad instead. The chatbot repeated the joke question and the user asked to close their bank account. The chatbot kept asking the same question until finally revealing the classic punchline - the chicken crossed the road to get to the other side.`
+
+1. After showing the summary to the user, it then initiates a transfer to a human agent. Unless you have configured a contact center, this will not work, but it is included in the kit to show how it would be invoked if you did have one configured.  See details in the next section.
+
+## Technical Details
+
+You can see how session history recording is enabled by contrasting the `Tell a joke` action with the `Tell a joke without recording context` action.  The assistant in the starter kit also has session history recording in the "No action matches" action (in the "Set by assistant" section of the actions list).  As you can see in the examples, you can enable this behavior in your own actions by applying the following changes:
+
+1. Add the `Record context` action to your assistant by copying the one in the kit.
+2. At each step in each action that you want to record:
+  - Set the `query_text` session variable to the text that the user says to the assistant (or empty string if there is no user input at that step)
+  - Set the `response` session variable to the the text that the assistant says to the user (or empty string if there is no assistant output at that step) -- you can then use that session variable in the "Assistant says" portion of the step so you are not repeating the same text twice.
+  - In the "And then" portion of the step, go to the `Record context` action and then continue after that action completes.
+  - If your existing step had something other than continuing to the next step in the "And then" portion, add a _new_ step that just does that additional "And then" behavior.  For example, step 3 in `Tell a joke without recording context` says "If you want to get the answer, you need to ask 'why'!" and then goes back to step 1.  When this is converted into `Tell a joke`, it gets split into two steps: the first says the same thing and records the context and the second goes back to step 1.
```

Comment:
again an image here is worth 1000 words

Reply:
I will do that in this PR or as a fast follow depending on timing.

Reply:
Done